molgenis-data-security/src/main/java/org/molgenis/data/security/user/UserServiceImpl.java

```diff
@@ -11,18 +11,17 @@
 import java.util.stream.Stream;
 
 import static java.util.stream.Collectors.toList;
-import static org.molgenis.data.security.auth.GroupMemberMetaData.GROUP_MEMBER;
 import static org.molgenis.data.security.auth.UserMetaData.USER;
 
 /**
- * Manage user in groups
+ * Manage user
```

Comment:
Remove superfluous comment

Reply:
Bit peculiar that we have two services that claim to do the same thing.